src/components/navigation-header/navigation-header.module.css

```diff
@@ -69,7 +69,7 @@
   display: flex;
   justify-content: left;
   width: calc(
-    var(--dev-dot-sidebar-width) + 2 * var(--main-area-padding-left-right) -
+    var(--dev-dot-sidebar-width) + var(--main-area-padding-left) -
```

Comment:
This maintains alignment of the nav header items on desktop with the main content area.

![CleanShot 2022-05-12 at 16 28 25@2x](https://user-images.githubusercontent.com/4624598/168177864-35690098-0670-4741-9aa0-c36a462f8ddf.png)

Note on the previous approach for context:
- `--main-area-padding-left` was always set to `24px`
- on mobile & tablet, `.main` had `0` padding-left, while `.mainArea` had `--main-area-padding-left`, resulting in a net padding-left of `24px`
- on desktop, _where this rule is relevant for alignment_, `.main` and `.mainArea` _both_ had padding-left set to `--main-area-padding-left`. This resulted in total padding-left of `48px`, creating the need to double the value here to achieve correct alignment.

In this PR, I've lightly refactored the use of `padding-left` to try to simplify the model:
- `--main-area-padding-left` is set to `24px` on mobile & tablet, and changes to `48px` on desktop
- so this rule can use the value directly rather than doubling it